package.json

```diff
@@ -21,11 +21,12 @@
   "dependencies": {
     "addressit": "git://github.com/dianashk/addressit.git#temp",
     "extend": "^3.0.0",
-    "lodash": "^4.11.1",
+    "lodash": "^4.13.1",
+    "node-postal": "openvenues/node-postal",
```

Comment:
best to lock it down to a specific tag or revision hash in case things change in the future, since we don't control that repo and won't know when breaking updates happen.